extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -584,4 +815,79 @@ public TopicPartition apply(Integer n)
         )
     );
   }
+
+  private Set<Integer> assignPartitionsAndSeekToStart(KafkaConsumer consumer, String topic)
```

Comment:
`assignPartitionsAndSeekToNext`?

When I first read this method name I thought it was going to be seeking to the startOffsets.